src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -1010,43 +992,28 @@ namespace
         }
     }
 
-    void transform_to_srgb(Image& image)
+    void transform_to_srgb(Image *image)
```

Comment:
Why these changes? Please revert.

Reply:
For the function `write_exr_image`, I move it into `write_image`.